services/tenant-ui/frontend/src/store/tenantStore.ts

```diff
@@ -27,6 +27,109 @@ import { useTokenStore } from './tokenStore';
 import { fetchItem } from './utils/fetchItem';
 import { fetchList } from './utils';
 
+interface ServerConfig {
```

Comment:
Mind moving this to types\index.ts and importing from there? Otherwise looking all good

Reply:
Sure thing